src/test/java/seedu/duke/command/AddNoteCommandTest.java

```diff
@@ -2,7 +2,7 @@
 
 import org.junit.jupiter.api.Test;
 
-class AddCommandTest {
+class AddNoteCommandTest {
```

Comment:
Same as before, please remind the person for AddEventCommandTest

Reply:
Already informed